src/Features/CSharp/Portable/InlineParameterNameHints/CSharpInlineParameterNameHintsService.cs

```diff
@@ -76,6 +76,16 @@ public CSharpInlineParameterNameHintsService()
         /// <returns>true when the adornment should be added</returns>
         private static bool IsExpressionWithNoName(ExpressionSyntax arg)
         {
+            if (arg.Parent.Parent is BracketedArgumentListSyntax bracketList)
+            {
+                // Do not need to add hints for array accesses if the length is 1
```

Comment:
The nittiest of nits: It sounds like this is saying we don't need hints if the length of the array is 1, but its actually the number of dimensions.

But it is the length of the `bracketList.Arguments` so I might be being pedantic :)